fragments/form/template.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <themecleanflex-components-block v-bind:model="model">
+    <div v-bind:data-per-path="model.path">
+      <div id="formFailure">
+        <p data-per-inline="failureText">{{failureText}}</p>
+      </div>
+      <form v-on:submit="defaultSubmit">
+        <vue-form-generator v-bind:model="formModel" v-bind:schema="schema" v-bind:options="formOptions"></vue-form-generator>
+        <input type="submit">
+      </form>
+    </div>
+  </themecleanflex-components-block>
+</template>
+
+<script>
+export default {
+    props: ['model'],
+    data() {
+      return {
+        formModel: {},
+        formOptions: {
+          validateAfterChange: true
+        },
+        failureText: ''
+      }
+    },
+    methods: {
+      defaultSubmit(e) {
+        e.preventDefault()
+        if(this.model.submitfunction != 'defaultSubmit' && this.model.submitfunction != '') {
+          if(window[this.model.submitfunction]) {
+            window[this.model.submitfunction](this.model,this.formModel)
+          } else {
+            const objs = this.model.submitfunction.split('.')
+            let parent = window
+            let i = 0
+            let found = false
+            while(!found && i < objs.length-1 && parent[objs[i]]) {
+              parent = parent[objs[i]]
+              if( i+1 == objs.length-1 && parent[objs[i+1]] ) {
+                found = true
```

Comment:
could we not just move the execution of the submit function to this location and use break to exit the while loop after it's executed?

Reply:
Fixed, new loop runs function from inside.